drivers/i2s/i2s_mcux_sai.c

```diff
@@ -308,11 +308,23 @@ static void i2s_dma_tx_callback(const struct device *dma_dev, void *arg, uint32_
 		/* TX queue has drained */
 		strm->state = I2S_STATE_READY;
 		LOG_DBG("TX stream has stopped");
+		goto disabled_exit_no_drop;
 	} else {
```

Comment:
I think the else might not be necessary, should we remove it to reduce nesting ?

Reply:
yes, we can remove it.